docs/releases/2.0.txt

```diff
@@ -248,6 +248,11 @@ Database backend API
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
 
+* The ``BaseDatabaseSchemaEditor._create_index_name`` method signature was
```

Comment:
The first argument of SchemaEditor._create_index_name() is now table_name rather than model.